app/controllers/AiModelController.scala

```diff
@@ -156,7 +156,8 @@ class AiModelController @Inject()(
           .submitJob(jobCommand, commandArgs, request.identity, dataStore.name) ?~> "job.couldNotRunTrainModel"
         newAiModel = AiModel(
           _id = modelId,
-          _organization = request.identity._organization,
+          _owningOrganization = request.identity._organization,
+          _organizations = List(request.identity._organization),
```

Comment:
Should probably also use the term sharedOrganizations that you use in code elsewhere